src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo;
+  onClose: () => void
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, onClose }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    async function userFetch() {
+      const loadedUser = await getUser(todo.userId);
+
+      setUser(loadedUser);
+    }
```

Comment:
Move to variable this function or just  move out from useEffect and here only call it

Reply:
done